### PR TITLE
Batch proofreading with LanguageTool: item descriptions

### PR DESCRIPTION
**Pull request by int-ua**
#### Summary
```SUMMARY: Content "Fixed many typos and mistakes in item descriptions after feeding them to LanguageTool"```

#### Describe the solution
https://discourse.cataclysmdda.org/t/mass-proofreading-with-languagetool-should-i-do-a-pr-care-to-help-with-converting-lt-suggestions-into-patches/21730

#### Testing
None

### CODE REVIEW

**Comment by kevingranade** [comment]
This pull request has been mentioned on **Cataclysm: Dark Days Ahead**. There might be relevant details there:

https://discourse.cataclysmdda.org/t/mass-proofreading-with-languagetool-should-i-do-a-pr-care-to-help-with-converting-lt-suggestions-into-patches/21730/4

File: data/json/items/tool/tailoring.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "type": "GENERIC",
     "category": "tools",
     "name": "bone sewing awl",
-    "description": "This is a crude sharpened bone awl; those were used for leatherworking before metal was discovered.  It can also serve as an improvised stabbing weapon, but will break quickly.",
```
Comment (int-ua) [inline data/json/items/tool/tailoring.json]: hmm

File: data/json/items/tool/tailoring.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
...
+    "description": "This is a crudely sharpened bone awl; those were used for leatherworking before metal was discovered.  It can also serve as an improvised stabbing weapon, but will break quickly.",
```
Comment (int-ua) [inline data/json/items/tool/tailoring.json]: hmm

Comment (ifreund) [reply data/json/items/tool/tailoring.json]: it's either `crude, sharpened` or `crudely sharpened`. I think the former is mostly likely intended.

File: data/json/items/gun/shot.json [hunk 3]
```diff
@@ -160,7 +160,7 @@
     "copy-from": "mossberg_500",
     "type": "GUN",
     "name": { "str": "Mossberg 500 Security", "str_pl": "Mossberg 500 Security" },
-    "description": "The Mossberg 500 is a popular series of pump-action shotguns, often acquired for military use.  It is noted for its high durability and low recoil.  This one is fitted with an 18.5 inch barrel.",
+    "description": "The Mossberg 500 is a popular series of pump-action shotguns, often acquired for military use.  It is noted for its high durability and low recoil.  This one is fitted with an 18.5-inch barrel.",
```
Comment (I-am-Erk) [inline data/json/items/gun/shot.json]:
```suggestion
    "description": "The Mossberg 500 is a popular series of pump-action shotguns, often acquired for military use.  It is noted for its high durability and low recoil.  This one is fitted with an 18.5 inch barrel.",
```

There are a number of entries below where "## unit" was changed to "##-unit". This is debatably correct at best and I don't think it should be changed.

File: data/json/items/gun/shot.json [hunk 4]
```diff
@@ -277,7 +277,7 @@
     "//": "Tileset whitelist for shotguns.",
     "type": "GUN",
     "name": { "str": "Remington 870 Wingmaster", "str_pl": "Remington 870 Wingmaster" },
-    "description": "With over 10 million made, the Remington 870 is one of the most popular shotguns on the market, and finds use with hunters and law enforcement agencies alike thanks to its high accuracy and muzzle velocity.  This one is a 28 inch barreled model for hunting fowl and game.",
+    "description": "With over 10 million made, the Remington 870 is one of the most popular shotguns on the market, and finds use with hunters and law enforcement agencies alike thanks to its high accuracy and muzzle velocity.  This one is a 28inch barreled model for hunting fowl and game.",
```
Comment (I-am-Erk) [inline data/json/items/gun/shot.json]:
```suggestion
    "description": "With over 10 million made, the Remington 870 is one of the most popular shotguns on the market, and finds use with hunters and law enforcement agencies alike thanks to its high accuracy and muzzle velocity.  This one is a 28 inch barreled model for hunting fowl and game.",
```

File: data/json/items/comestibles/raw_veggy.json [hunk 5]
```diff
@@ -302,7 +302,7 @@
     "comestible_type": "FOOD",
     "symbol": "%",
     "calories": 109,
-    "description": "An herb with a small purple flower.  The roots are nutritious, but tough and bland until cooked.",
+    "description": "A herb with a small purple flower.  The roots are nutritious, but tough and bland until cooked.",
     "//": "Serving Size is 1 cup of sliced roots. Salsify grows in bountiful clumps, it's a weed.",
```
[the remark this answers is not in the document]
Comment (int-ua) [reply data/json/items/comestibles/raw_veggy.json]: > An herb
That was not intended, right?
Can you please comment on why we should remove the comment while I'm doing it?

[the remark this answers is not in the document]
Comment (ifreund) [reply data/json/items/comestibles/raw_veggy.json]: "An herb" is correct here, I think the comment was removed by accident

File: data/json/items/armor/jewelry.json [hunk 6]
```diff
@@ -1379,7 +1379,7 @@
     "material": [ "plastic" ],
     "price_postapoc": 30,
     "weight": "10 g",
-    "description": "A star shaped badge, on it you can read \"Foodkid\" in bold letters.  This is Foodperson's trusty sidekick's badge."
+    "description": "A star shaped badge. On it, you can read \"Foodkid\" in bold letters.  This is Foodperson's trusty sidekick's badge."
```
Comment (ZhilkinSerg) [inline data/json/items/armor/jewelry.json]:
```suggestion
    "description": "A star shaped badge.  On it, you can read \"Foodkid\" in bold letters.  This is Foodperson's trusty sidekick's badge."
```

Comment (int-ua) [reply data/json/items/armor/jewelry.json]: Two spaces? Why?

Comment (ZhilkinSerg) [reply data/json/items/armor/jewelry.json]: https://travis-ci.org/CleverRaven/Cataclysm-DDA/jobs/651628210#L4684

https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/MANUAL_OF_STYLE.md

**Pull request merged**